react/AddToCartButton.tsx

```diff
@@ -157,15 +165,15 @@ const AddToCartButton: FC<Props> = ({
   )
 
   const unavailableButtonContent = (
-    <FormattedMessage id="store/buyButton-label-unavailable">
+    <FormattedMessage id="store/add-to-cart.label-unavailable">
       {message => <span className={handles.buttonText}>{message}</span>}
     </FormattedMessage>
   )
 
   const tooltipLabel = (
-    <FormattedMessage id="store/add-to-cart.select-sku-variations">
-      {message => <span className={handles.errorMessage}>{message}</span>}
-    </FormattedMessage>
+    <span className={handles.tooltipLabelText}>
+      {intl.formatMessage(messages.skuVariations)}
```

Comment:
Messages is not dealing with static objects inside formatMessage too

example:
```jsx
intl.formatMessage({ id: "foobar" })
```